api/capi/src/nnstreamer-capi-util.c

```diff
@@ -85,15 +85,16 @@ ml_tensors_info_destroy (ml_tensors_info_h info)
 {
   ml_tensors_info_s *tensors_info;
 
+  if (!info)
+    return ML_ERROR_NONE;
+
   check_feature_state ();
 
   tensors_info = (ml_tensors_info_s *) info;
 
-  if (!tensors_info)
-    return ML_ERROR_INVALID_PARAMETER;
-
   ml_tensors_info_free (tensors_info);
   g_free (tensors_info);
+  info = NULL;
```

Comment:
No effect?

Reply:
initializing the info/data as `NULL` is better than let it go as an unknown state.

Reply:
Because the argument is ```ml_tensors_info_h info```, ```info=NULL;``` has no effect. It needs to be ```ml_tensors_info_h *info``` and ```*info = NULL;``` to have effects.